shared/src/androidMain/kotlin/org/jetbrains/kotlinconf/utils/StoreUrl.android.kt

```diff
@@ -0,0 +1,3 @@
+package org.jetbrains.kotlinconf.utils
+
+actual fun getStoreUrl(): String? = "https://play.google.com/store/apps/details?id=com.jetbrains.kotlinconf"
```

Comment:
Should this also be part of the resources?

Reply:
Like in string resources? I think things that won't be localized don't necessarily have to be extracted there.

(I don't believe we're doing localization at all, but that's the purpose of string resources to begin with.)

Reply:
I'm not against keeping it in string, but having all constants in one place in generally is a good practice

Reply:
I'll keep it like this for now, as these don't really have to be in common code, and if they're in different source sets then they won't be in one place anyway